### PR TITLE
Replacing learning_rate_schedule with keras.optimizers.schedules 

### PR DESCRIPTION
**Pull request by abhilash1910**
This is an address to the bug in tf raised at : https://github.com/tensorflow/tensorflow/issues/55510 
Following PR :https://github.com/tensorflow/tensorflow/pull/55593
#55593
@gbaned  @fchollet @qlzh727

### CODE REVIEW

File: keras/optimizers/optimizer_v2/optimizer_v2.py [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import contextlib
 import functools
 import warnings
+import keras.optimizers.schedules
```
Comment (fchollet) [inline keras/optimizers/optimizer_v2/optimizer_v2.py:23]: This symbol is unused.

File: keras/optimizers/optimizer_v2/optimizer_v2.py [hunk 2]
```diff
@@ -792,8 +793,8 @@ def _set_hyper(self, name, value):
       if (callable(prev_value)
           or isinstance(prev_value,
                         (tf.Tensor, int, float,
-                         learning_rate_schedule.LearningRateSchedule))
-          or isinstance(value, learning_rate_schedule.LearningRateSchedule)):
+                         keras.optimizers.schedules.LearningRateSchedule))
```
Comment (fchollet) [inline keras/optimizers/optimizer_v2/optimizer_v2.py:796]: Fix indent.

File: keras/optimizers/optimizer_v2/optimizer_v2.py (hunk 1 above)
Comment (abhilash1910) [reply keras/optimizers/optimizer_v2/optimizer_v2.py]: Resolved

File: keras/optimizers/optimizer_v2/optimizer_v2.py (hunk 2 above)
Comment (abhilash1910) [reply keras/optimizers/optimizer_v2/optimizer_v2.py:795]: Line 796 has keras.optimizers.scheduler as the only change , indent has remained the same. Could you please suggest on this ?  I can convert it to : (tf.Tensor, int, float, keras.optimizers.schedules.LearningRateSchedule)) in a single line but that would hamper the preallocated tabs present in the code.

**Review by fchollet — APPROVED** [review]
LGTM, thanks

File: keras/optimizers/optimizer_v2/optimizer_v2.py [hunk 3]
```diff
@@ -792,8 +792,8 @@ def _set_hyper(self, name, value):
       if (callable(prev_value)
           or isinstance(prev_value,
                         (tf.Tensor, int, float,
-                         learning_rate_schedule.LearningRateSchedule))
-          or isinstance(value, learning_rate_schedule.LearningRateSchedule)):
+                        keras.optimizers.schedules.LearningRateSchedule))
```
Comment (old-school-kid) [inline keras/optimizers/optimizer_v2/optimizer_v2.py:795]: keras not imported

File: keras/optimizers/optimizer_v2/optimizer_v2.py [hunk 4]
```diff
@@ -792,8 +792,8 @@ def _set_hyper(self, name, value):
...
+          or isinstance(value, keras.optimizers.schedules.LearningRateSchedule)):
```
Comment (old-school-kid) [inline keras/optimizers/optimizer_v2/optimizer_v2.py:796]: same error

File: keras/optimizers/optimizer_v2/optimizer_v2.py [hunk 5]
```diff
@@ -802,7 +802,7 @@ def _get_hyper(self, name, dtype=None):
     if not self._hypers_created:
       self._create_hypers()
     value = self._hyper[name]
-    if isinstance(value, learning_rate_schedule.LearningRateSchedule):
+    if isinstance(value, keras.optimizers.schedules.LearningRateSchedule):
```
Comment (old-school-kid) [inline keras/optimizers/optimizer_v2/optimizer_v2.py:805]: same error

File: keras/optimizers/optimizer_v2/optimizer_v2.py [hunk 6]
```diff
@@ -1051,7 +1051,7 @@ def iterations(self, variable):
   def _decayed_lr(self, var_dtype):
     """Get decayed learning rate as a Tensor with dtype=var_dtype."""
     lr_t = self._get_hyper("learning_rate", var_dtype)
-    if isinstance(lr_t, learning_rate_schedule.LearningRateSchedule):
+    if isinstance(lr_t, keras.optimizers.schedules.LearningRateSchedule):
```
Comment (old-school-kid) [inline keras/optimizers/optimizer_v2/optimizer_v2.py:1054]: same error

**Comment by abhilash1910** [comment]
👍🏻 thanks @fchollet

**Comment by chenmoneygithub** [comment]
I don't think this change work? `learning_rate_schedule.LearningRateSchedule` is an alias to `keras.optimizers.schedules.LearningRateSchedule` ([export line](https://github.com/keras-team/keras/blob/v2.8.0/keras/optimizer_v2/learning_rate_schedule.py#L26)), so basically nothing is changed here. 

I checked the original error, and it is caused by referring to the keras optimizer code under Tensorflow repo `tensorflow/python/keras/optimizer_v2/optimizer_v2.py`, which is a deprecated path.

**Comment by abhilash1910** [comment]
@chenmoneygithub So should that change be addressed in tf ? I saw that in keras it is indeed an alias.

**Comment by fchollet** [comment]
@chenmoneygithub good catch, thank you!

@abhilash1910 any reference to `tensorflow/python/keras` code anywhere should be removed. The `keras` package should be used instead.

**Comment by south-ocean** [comment]
> @chenmoneygithub good catch, thank you!
> 
> @abhilash1910 any reference to `tensorflow/python/keras` code anywhere should be removed. The `keras` package should be used instead.

So how to resolve this problem? if i don't change files follow these changes, the error also ocur in tf2.7.

**Comment by JiaqingFu** [comment]
@chenmoneygithub @fchollet how can I solve the problem with a stable released tf2.6，we are required to develop algorithm with tf2.6
